src/app/actions/SearchActions.js

```diff
@@ -60,25 +61,26 @@ export const searchPost = (query, field) => {
     });
 };
 
-export const fetchWork = workId => dispatch =>
-  axios.get(recordUrl, { params: { identifier: workId } })
-    .then((resp) => {
-      if (resp.data) {
-        dispatch(workDetail(resp.data));
-      }
-    })
-    .catch((error) => {
-      console.log('An error occurred during fetchWork', error.message);
-      throw new Error('An error occurred during fetchWork', error.message);
-    });
+export const fetchWork = workId => dispatch => axios
```

Comment:
I think having `axios` on the next line makes it much more readable. This is very nice to see it is understandable but it's not easily readable. I'm not sure how you and Greg prefer but I prefer quick readability.